_docs/Qabel-Client-Local-Data.md

```diff
@@ -6,6 +6,18 @@ title: Local Data
 ## Abstract
 Information and configuration settings stored on each client.
 
+## Data Types
+The types and format of data which is exchanged or can be exported and imported are defined as follows:
+
+| Data type | Definition |
+| --- | --- |
+| INT | 32-bit signed two's complement integer  |
+| LONG | 64-bit signed two's complement integer |
+| STR | UTF-8 encoded and (if applicable) null terminated |
+| KEY | Hex-String with ASCII encoding. Every byte of data is converted into the corresponding 2-digit hex representation. The returned bytes object is therefore twice as long as the length of data.|
+| URL | ASCII encoded URIs according to [RFC 3986](http://tools.ietf.org/html/rfc3986)), for Drop URL see [here](../Qabel-Protocol-Drop#url) |
+| UUID | 128 bit random number ASCII encoded Universally Unique IDentifier, see [this definition](https://en.wikipedia.org/wiki/Universally_unique_identifier#Definition) |
```

Comment:
Applied the wording from the RFC https://tools.ietf.org/html/rfc4122, should I change it anyway?